src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -439,7 +443,7 @@ def __init__(self, scope: core.Construct, id: str, map_params: dict, ssm_params:
             ),
             "topic_arn": map_params.get('topic_arn'),
             "name": map_params['name'],
-            "completion_trigger": map_params.get('completion_trigger'),
+            "completion_trigger": map_params.get('completion_trigger', None) or map_params.get("triggers", {}).get("on_complete", None),
```

Comment:
Since we prefer the use of `triggers/on_complete`, I think we should always that one when both are defined at the same time.

```suggestion
            "completion_trigger": map_params.get('triggers', {}).get('on_complete', map_params.get('completion_trigger')),
```